src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/link-apm-applications-kubernetes.mdx

```diff
@@ -76,7 +76,7 @@ nri-metadata-injection:
     enabled: true
 ```
 
-After deploying the webhook, restart your application pods. They need to pick up the required environment variables.
+**Important:** Your application pods will need to be restarted after the webhook is deployed, so that they can pick up the necessary environment variables.
```

Comment:
Should we say nri-metadata-injection or the K8s integration instead of webhook?
Some customers may not what a webhook is or from where this webhook comes from